src/pyhf/infer/test_statistics.py

```diff
@@ -2,6 +2,48 @@
 from .mle import fixed_poi_fit, fit
 from ..exceptions import UnspecifiedPOI
 
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def _qmu_like(mu, data, pdf, init_pars, par_bounds):
+    """
+    Clipped version of _tmu where _qmu = 0 if muhat > 0 else _tmu
+
+    If the lower bound of the POI is 0 this automatically implments
+    qmu_tilde. Otherwise this is qmu (no tilde).
+    """
+    tensorlib, optimizer = get_backend()
+    tmu_stat, (mubhathat, muhatbhat) = _tmu_like(
+        mu, data, pdf, init_pars, par_bounds, return_fitted_pars=True
+    )
+    qmu = tensorlib.where(
+        muhatbhat[pdf.config.poi_index] > mu, tensorlib.astensor(0.0), tmu_stat
+    )
+    return qmu
```

Comment:
```suggestion
    qmu_like = tensorlib.where(
        muhatbhat[pdf.config.poi_index] > mu, tensorlib.astensor(0.0), tmu_stat
    )
    return qmu_like
```

To avoid confusion, given that this can return `qmu` and `qmu_tilde` should we make this explicit in the variable names?